roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,83 @@
+"""
+Module for performing checks on an Fluentd logging deployment configuration
+"""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class FluentdConfig(OpenShiftCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    logging_namespace = None
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return super(FluentdConfig, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        """Check that Fluentd has running pods, and that its logging config matches Docker's logging config."""
+
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        config_error = self.check_logging_config(task_vars)
+        if config_error:
+            msg = ("The following Fluentd logging configuration problem was found:"
+                   "\n-------\n"
+                   "{}".format(config_error))
+            return {"failed": True, "changed": False, "msg": msg}
+
+        return {"failed": False, "changed": False, "msg": 'No problems found with Fluentd logging configuration.'}
+
+    def check_logging_config(self, task_vars):
+        """Ensure that the configured Docker logging driver matches fluentd settings.
+        This means that, at least for now, if the following condition is met:
+
+            openshift_logging_fluentd_use_journal == True
+
+        then the value of the configured Docker logging driver should be "journald".
+        Otherwise, the value of the Docker logging driver should be "json-file".
+        Returns an error string if the above condition is not met, or None otherwise."""
+        use_journald = get_var(task_vars, "openshift_logging_fluentd_use_journal", default=True)
```

Comment:
May or may not have any consequence to this PR, but in any case, FWIW I see this variable was deprecated 14 days ago:

- [roles/openshift_logging/README.md#L55](https://github.com/openshift/openshift-ansible/blame/d13c11aceb2c2ccf2c1db1ead9f545d7b0891cfc/roles/openshift_logging/README.md#L55)
- [roles/openshift_logging_fluentd/tasks/main.yaml#L18-L20](https://github.com/openshift/openshift-ansible/blob/d13c11aceb2c2ccf2c1db1ead9f545d7b0891cfc/roles/openshift_logging_fluentd/tasks/main.yaml#L18-L20)